main.py

```diff
@@ -0,0 +1,37 @@
+#importing the necessary libraries
+import csv
+
+#creating the variables to store the individual values
+temperature = []
+humidity = []
+
+#to open the csv file
+with open("weather_data.csv", "r") as csvfile:
+    reader = csv.reader(csvfile)
+    next(reader)  #to skip the header row
+
+    for row in reader:
+        temperature.append(float(row[1]))
+        humidity.append(int(row[2]))
+
+# #to sort the temperature and calculate the max and min
+# sorted_temperature = sorted(temperature)
+# max_temperature = sorted_temperature[-1]
+# min_temperature = sorted_temperature[0]
+# print(max_temperature)
+# print(min_temperature)
+
+
+
+#to calculate the average of temperature and humidity
+avg_temperature = sum(temperature) / len(temperature)
+avg_humidity = sum(humidity) / len(humidity)
```

Comment:
This is essentially the same operation twice - is there a way you could DRY (Don't Repeat Yourself) this code up a bit? This is obviously a pretty small example, but best to avoid duplication where we can!